mbedtls/patches/03_rename_debug_var.patch

```diff
@@ -0,0 +1,40 @@
+diff -ur '--exclude=*.git' mbedtls-2.28.0/programs/ssl/ssl_context_info.c mbedtls-2.28.0-phoenix/programs/ssl/ssl_context_info.c
+--- mbedtls-2.28.0/programs/ssl/ssl_context_info.c	2022-05-12 15:49:04.327065471 +0200
++++ mbedtls-2.28.0-phoenix/programs/ssl/ssl_context_info.c	2022-05-12 15:44:21.216459932 +0200
+@@ -118,7 +118,8 @@
+ FILE *b64_file = NULL;                  /* file with base64 codes to deserialize */
+ char conf_keep_peer_certificate = 1;    /* MBEDTLS_SSL_KEEP_PEER_CERTIFICATE from mbedTLS configuration */
+ char conf_dtls_proto = 1;               /* MBEDTLS_SSL_PROTO_DTLS from mbedTLS configuration */
+-char debug = 0;                         /* flag for debug messages */
++/* 'debug' has been already defined as global variable in Phoenix-RTOS */
```

Comment:
`debug()` is a syscall, not a global variable.

I don't know, why this name conflict happens to be honest. mbedtls does not include `<sys/debug.h>`. Verify if this is really necessary.

Reply:
Yeah , I will change it to syscall. I was looking for including something related to syscalls in mbedtls and the keyword `syscall` appears only in binaries of mbedtls programs. Couldn't find anything more about it

Reply:
I'm guessing but this is non-static global variable so has **external** linkage by default. This results in _multiple definitions of symbol `debug`_ in linking stage.

better solution would be to prepend the variable declaration with `static` keyword if that's the case

Reply:
`static` solves the problem, I agree on that it's a better solution